### DISCUSSION TITLE
v1.0.0-beta8-fix1

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#1167] [General]
> 注意: 此版本为开发者预览版, 不建议用于生产环境

新增: 子频道信息更新事件. 
新增: 子频道创建事件. 
新增: 子频道删除事件. 
新增: 频道消息支持 `[CQ:at]` 了. 
新增: 支持通过配置文件调整控制台日志染色了. by @Akegarasu 

修复: 在特殊情况下, 发送频道信息时可能会报错并断开WS连接的问题.
修复: BOT加入了不包含管理组频道的情况下, 拉取频道列表会因错误中断导致频道拉取不全的问题. 
修复: 频道消息在 `array` 格式上报的情况下会丢失 `image` 类型消息的问题. by @Akegarasu 
修复: `HTTP POST` 忘记提交 `body` 的问题. by @wdvxdr1123 
修复: 通过 `base64` 向 `websocket`  发送大图时可能会遇到读取字节数限制的问题. by @wdvxdr1123 

其他: 优化了长消息服务器连接错误时的提示信息, 并为腾讯云服务器添加了DNS修改警告.
其他: 删除了 `[CQ:gift]` 的支持. by @wdvxdr1123 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-beta8-fix1'>v1.0.0-beta8-fix1</a>.</em>

**Comment by xsclyf** [comment]
好像通过外网ip地址访问API接口会被拒绝访问emmm

**Reply by Ink-33** [reply]
你监听的host是0.0.0.0吗

**Reply by xsclyf** [reply]
emmm默认的127.0.0.1

**Reply by xsclyf** [reply]
玩分感谢……监听0.0.0.0就可以了，(:з」∠)
